css/style.css

```diff
@@ -13,6 +13,13 @@ ul li {
 .hide {
   display: none;
 }
+
+.show-less {
+  display: none;
+}
+#load-more {
```

Comment:
Shouldn't `load-more` be a class?

Reply:
I had forgotten to remove these styling even though they were no longer in use. I have now deleted these lines of code to clean up the styling.